vpr/src/route/router_lookahead_extended_map.cpp

```diff
@@ -230,7 +230,7 @@ std::pair<float, float> ExtendedMapLookahead::get_expected_delay_and_cong(int in
 
     VTR_LOGV_DEBUG(f_router_debug, "Requested lookahead from node %d to %d\n", size_t(from_node), size_t(to_node));
     const std::string& segment_name = device_ctx.rr_segments[from_seg_index].name;
-    VTR_LOGV_DEBUG(f_router_debug, "Lookahead returned %s (%d) with distance (%zd, %zd)\n",
+    VTR_LOGV_DEBUG(f_router_debug, "Lookahead returned %s (%d) with distance (%d, %d)\n",
```

Comment:
This fix belongs upstream too, yes?

Reply:
Indeed, opening a PR now

Reply:
Added to the extended lookahead map followup PR: https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1536